docs/packages.docs.config

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="mdoc" version="5.6.1"  />
-</packages>
```

Comment:
I removed this file, is it still needed?

Reply:
@mattleibow correct, it is no longer needed ... since you're pulling mdoc through msbuild :)